include/variables.h

```diff
@@ -1644,8 +1644,8 @@ extern UNK_PTR D_801D1540;
 // extern f32 sFactorialTbl[13];
 extern Vec3f D_801D15B0;
 extern Vec3s D_801D15BC;
-extern Mtx D_801D1DE0;
-extern MtxF D_801D1E20;
+extern Mtx gMtxClear;
+extern MtxF gMtxFClear;
```

Comment:
fyi this will conflict with my name of `gIdentityMtx`/`gIdentityMtxF` in Bigslime, so we should probably resolve it at some point

Reply:
From what I remember, we didn't get a satisfactory resolution about the naming of those variables.
I think we should keep the `gMtxClear` and `gMtxFClear` to keep the parity with OoT at least for now.
Tell me if you have any other idea

Reply:
After the discussion in the Discord, I'll rename those two to `gIdentityMtx`/`gIdentityMtxF`